R/run.R

```diff
@@ -555,6 +560,7 @@ CmdStanProcs <- R6::R6Class(
       private$proc_total_time_ = zeros
       private$show_stderr_messages_ = show_stderr_messages
       private$show_stdout_messages_ = show_stdout_messages
+      private$iter_done_ = as.list(rep(0,length(private$proc_ids_)))
```

Comment:
```suggestion
      private$iter_done_ <- as.list(rep(0,length(private$proc_ids_)))
```

For code consistency we should use `<-`, although it looks like the few lines above this also use `=` (an oversight, we should fix those too)